assembly/assembly-main/src/assembly/bin/docker.sh

```diff
@@ -259,7 +267,7 @@ check_docker
 init
 
 # run application
-"${CHE_HOME}"/bin/che.sh run &
+sudo -u $CHE_USER -E sh -c "${CHE_HOME}/bin/che.sh run" &
```

Comment:
Have you tested if Che can be cleanly stopped? For example, when stopping Che, running workspaces (if they exist) are stopped too? I'm wondering that because SIGTERM signals are usually not well propagated when wrapping the command with a shell (i.e. `sh -c`).